admin_scripts/os2displaypc/auto_activate_chromium.py

```diff
@@ -92,16 +119,17 @@
         extracted_filepath)
     )
 else:
-     print('Chromedriver {} is already setup.'.format(driver_version))
+    print('Chromedriver {} is already setup.'.format(driver_version))
```

Comment:
I have a question about all of this logic: is it necessary? :-)

In particular, the `chromium` snap seems to include `chromedriver` (see `/snap/chromium/*/usr/lib/chromium-browser/chromedriver`). Can't we just use that directly?

Reply:
I only removed the preceding whitespace just to be clear not that you're saying otherwise, but if we could avoid all that mess of downloading the correct version of the driver - which has to be updated when every new major version is released (and I seem to recall other scripts work similarly, at least the work in progress on that Aula script) - it would be great!
I also considered downloading some portable chromium version to get out of that problem, but if the chromedriver is downloaded and updated automatically now that seems even better!